src/locales/intl.ts

```diff
@@ -274,7 +274,17 @@ export const trimEnd = (val: string): string => {
 
 // Return a formatted percent string based on a number or string that is < 1.0
 // and greater than -1.0
-export const toPercentString = (percentVal: string | number, options?: IntlNumberFormatOptionsType): string => {
+export const toPercentString = ({
+  percentVal,
+  intlOpts,
+  maxPrecision = 1,
+  minPrecision = 0
+}: {
+  percentVal: string | number
+  maxPrecision?: number
+  minPrecision?: number
+  intlOpts?: IntlNumberFormatOptionsType
+}): string => {
```

Comment:
I think the value to convert should remain as the first parameter. It's the only non-optional "option", and would never be omitted.

I also dislike destructuring in the argument list, especially with default values, since we are mixing interface & implementation. Maybe do:

```ts
export const toPercentString = (percentVal: string | number, opts: {...}): string {
  const {
    intlOpts,
    maxPrecision = 1,
    minPrecision = 0
  } = opts
```

I think this would be easier to use & understand, but I am open to keeping it as-is if you have strong opinions otherwise.